scripts/session-trailer.sh

```
@@ -0,0 +1,27 @@
+# the trailer without a manual export.
+#
+# Seat resolution: the newest .state under the machine home is the seat
+# that last ran preflight — one live seat per checkout (herd.md seat
+# hygiene). No state file, or a missing named token: leave the env var
+# unset; the helper no-ops and commitlint names the missing trailer.
+
+msg="$1"
+[ -n "$msg" ] || exit 0
+if [ -z "${GANDR_SESSION_TOKEN:-}" ]; then
+  herd_home="${GANDR_HERD_HOME:-$HOME/.config/gandr-herd}"
+  state=$(ls -t "$herd_home"/sessions/*.state 2>/dev/null | sed -n 1p)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)